config/configtls/configtls.go

```diff
@@ -359,11 +362,18 @@ func (c Config) loadCertificate() (tls.Certificate, error) {
 		keyPem = []byte(c.KeyPem)
 	}
 
-	certificate, err := tls.X509KeyPair(certPem, keyPem)
-	if err != nil {
-		return tls.Certificate{}, fmt.Errorf("failed to load TLS cert and key PEMs: %w", err)
+	if c.TPMConfig.Enabled {
```

Comment:
Does TPM configuration need to be accounted for in the list of cases at the top of this function that returns an error if multiple certs are configured?

Reply:
no